backend/api/user.go

```diff
@@ -28,6 +28,15 @@ func user(c echo.Context) error {
 	return c.JSON(http.StatusOK, "User get handler")
 }
 
+func logout(c echo.Context) error {
+	// Setting MaxAge<0 means delete cookie now.
+	u := http.Cookie{
+		Name: "UserAuth"}
+	u.Expires = time.Unix(0, 0)
+	c.SetCookie(&u)
+	return c.JSON(http.StatusOK, "Old Cookie Deleted. Logged Out!\n")
+	// !!!!!!!!!!!Refersh Token STILL needs to be added to the database
```

Comment:
Maybe this comment could be a little more clear. Something like `// TODO: Blacklist refresh token`